framework/include/actions/ActionWarehouse.h

```diff
@@ -202,6 +202,8 @@ class ActionWarehouse : public ConsoleStreamInterface
    * various warehouse operations during the problem setup phase.
    * @param state Flag indicating whether to show action information.
    */
+  void showActionDependencies(bool state = true) { _show_action_dependencies = state; }
+
```

Comment:
Make doc strings appropriate, given that there is not one at the variable definition

Reply:
There is one docstring supposed for these three functions. I guess we need to add something like `///@{`.